docs/_data/nav_docs.yml

```diff
@@ -1,9 +1,81 @@
-- title: Tutorial
-- title: Basic Guides
+- title: Quick Start
   items:
-    - id: installation
-      title: Installation
-    - id: hello-world
-      title: Hello World
-- title: Advanced Guides
+  - id: getting-started
+    title: Getting Started
+  - id: thinking-in-react
+    title: Thinking in React
+- title: Community Resources
+  items:
+  - id: conferences
+    title: Conferences
+  - id: videos
+    title: Videos
+  - id: complementary-tools
+    title: Complementary Tools
+    href: https://github.com/facebook/react/wiki/Complementary-Tools
+  - id: examples
+    title: Examples
+    href: https://github.com/facebook/react/wiki/Examples
+- title: Guides
+  items:
+  - id: why-react
```

Comment:
What is the reason for adding these back in? I intentionally removed them from docs and left them in docs-old so that we tell new and old content apart and gradually migrate it.